src/object/srv_obj.c

```diff
@@ -931,6 +970,13 @@ obj_fetch_create_maps(crt_rpc_t *rpc, struct bio_desc *biod)
 	 * Will be freed in obj_rw_reply
 	 */
 	iods_nr = orw->orw_iod_array.oia_iod_nr;
+
+	/* Re-entry case. */
+	if (orwo->orw_maps.ca_count != 0) {
+		D_ASSERT(orwo->orw_maps.ca_count == iods_nr);
+		return 0;
```

Comment:
same, for re-entry case, the details of maps (determined by bio details) must be same as last time fetch?
not sure, just confirm

Reply:
Similar as above.